build.gradle

```diff
@@ -102,7 +103,7 @@ subprojects {
                 'io.dropwizard:dropwizard-auth:' + dependencyVersions.dropwizard,
                 'io.dropwizard:dropwizard-configuration:' + dependencyVersions.dropwizard,
                 'io.dropwizard:dropwizard-metrics-graphite:' + dependencyVersions.dropwizard,
-                'com.hubspot.dropwizard:dropwizard-guicier:0.9.1.0',
+                'com.hubspot.dropwizard:dropwizard-guicier:1.0.0.6',
```

Comment:
Please update dropwizard guicer to the latest version '1.0.9.1'.

Reply:
Silly me for assuming their `releases` page contained their releases https://github.com/HubSpot/dropwizard-guicier/releases !

Reply:
1.0.9.1 has build errors and it is outside the scope of this work to do the full upgrade. I will raise it as a tech-debt card for the relevant team

Reply:
There is 'com.hubspot.dropwizard:dropwizard-guicier:0.9.1.0' in the force section. Removing it breaks hub.